tests/system/action/poll/test_anonymize.py

```diff
@@ -45,3 +47,109 @@ def test_anonymize(self) -> None:
         assert user.get("vote_$1_ids") == []
         assert user.get("vote_delegated_vote_$_ids") == ["1"]
         assert user.get("vote_delegated_vote_$1_ids") == []
+
+
+class AnonymizePoll(BaseActionTestCase):
```

Comment:
What is the difference to the class above, `PollAnonymizeActionTest`? Just have one class for anonymizing and "squash" these two classes.

Reply:
Merged into one class. One test removed.